packages/modules/core/src/components/App.js

```diff
@@ -86,15 +98,53 @@ export const DigitApp = ({ stateCode, modules, appTenants, logoUrl }) => {
         </div>
       </Route>
       <Route path="/digit-ui/citizen">
-        <TopBar
-          img={cityDetails?.logoId}
-          ulb={`${t(cityDetails?.i18nKey)} ${ulbCamel(t("ULBGRADE_MUNICIPAL_CORPORATION"))}`}
-          isMobile={true}
-          toggleSidebar={() => toggleSidebar(!isSidebarOpen)}
-          logoUrl={logoUrl}
-          onLogout={handleLogout}
-          userDetails={userDetails}
-        />
+        {mobileView && (
+          <TopBar
+            img={cityDetails?.logoId}
+            ulb={`${t(cityDetails?.i18nKey)} ${ulbCamel(t("ULBGRADE_MUNICIPAL_CORPORATION"))}`}
+            isMobile={true}
+            toggleSidebar={() => toggleSidebar(!isSidebarOpen)}
+            logoUrl={logoUrl}
+            onLogout={handleLogout}
+            userDetails={userDetails}
+          />
+        )}
+        {!mobileView && (
+          <div className="topbar">
```

Comment:
Can we make a new react-component?

Reply:
Yes, we can but as we discussed that there may be a possibility that this will change as designs are not shared yet. 
once finalized we can move this code to a new component